instrumentation-genai/opentelemetry-instrumentation-google-genai/tests/common/otel_mocker.py

```diff
@@ -26,7 +26,7 @@
 from opentelemetry.sdk._events import EventLoggerProvider
 from opentelemetry.sdk._logs import LoggerProvider
 from opentelemetry.sdk._logs.export import (
-    InMemoryLogExporter,
+    InMemoryLogRecordExporter,
```

Comment:
I think you should handle the ImportError and alias the old one to the same name as the new one so we can merge this before the other one. Technically there's no constraint the the test would run against latest git outside of this CI

Reply:
I was thinking something like the following so we can merge this and tests will be also green on the core PR  with the rename:

```
try:
    from opentelemetry.sdk._logs.export import (
        InMemoryLogRecordExporter
    )
except ImportError:
    from opentelemetry.sdk._logs.export import (
        InMemoryLogExporter as InMemoryLogRecordExporter
    )
```

Said that I commented on the core PR with a proposal to keep the old names deprecated for a release cycle or so and then drop so it's more convenient for people needing to update code?